CS3230Project/ErrorMessages/ErrorMessages.cs

```diff
@@ -86,8 +86,8 @@ public static class PatientErrorMessages
         /// <summary>
         /// Error message for both first and last names being either null or empty
         /// </summary>
-        public static string FirstAndLastNamesCannotBothBeNullOrEmpty =
```

Comment:
That is a start but again there should be separate checks for null and empty

Reply:
That was changed because I changed the error message that is display in the window when searching by empty names. I have not attempted to fix this yet.